platforms/documentation/docs/src/docs/userguide/best-practices/best_practices_performance.adoc

```diff
@@ -42,3 +42,42 @@ NOTE: Do not rely on the default encoding of the underlying JVM or operating sys
 === Tags
 
 `<<tags_reference.adoc#tag:properties,#properties>>`, `<<tags_reference.adoc#tag:caching,#caching>>`
+
+[[use_build_cache]]
+== Use build cache
+
+Use the build cache to save time by reusing outputs produced by other builds.
+
+=== Explanation
+
+The build cache can retrieve outputs from any earlier build, avoiding unnecessary work which has already been done during a previous run.
+
+NOTE: Enabling the build cache introduces a slight overhead, especially on "cold" builds (i.e. when populating the cache for the first time). However, enabling it brings substantial benefits in the long run.
+
+=== Example
```

Comment:
This example should probably at least note that you can see the `FROM-CACHE` in the output of the compilation tasks run in the 'do' example.  Probably worth a short paragraph explaining that tasks are run multiple times, output cleaned from the buildDir in between, and work of compilation successfully avoided on the second run.